Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,283 @@
+
+***
+
+## Developers:
+
+* optimal media production (Until 2011)[^CopyX_Current_Website]
+* optimal media (Current)
+
+[^CopyX_Current_Website]: [The current optimal media website link.](https://www.optimal-media.com/)
+***
+
+## Overall Description:
+
+Ring protection copy protection scheme from german company optimal media[^CopyX_Original_Website]. As seen in Names, four different types seem to have been in use[^CopyX_Original_Website]. Note that it has not yet been confirmed whether light and professional correspond to the currently established distinct behaviors, but it seems to be the most likely conclusion.
+Had to be mastered at a specific optimal media production plant[^CopyX_2009_Document].
+
+[^CopyX_2009_Document]: [Optimal media document from 2009, seemingly for prospective customers.](http://press9.de/fileadmin/user_upload/PDF/copy_protection_july_09.pdf)
+***
+
+## Timeline: 
+
+#### 2000s
+
+* 2001?: Seems to come into existence about here.
+
+* 2009?: Some sort of overhaul occurs.
+
+***
+
+## Versions & Differences: 
+
+#### Tested:
+
+* copy-X light[^84759][^107929][^81628][^Kenny][^101786]
+* copy-X professional[^108150][^48393][^82475][^DE_EM2DX][^Petterson]
+
+[^84759]: [Black Cats and Pointed Hats (USA) (En,De)](http://redump.org/disc/84759/)
+[^107929]: [Robinson Crusoe (USA)](http://redump.org/disc/107929/)
+[^81628]: [Webmaster (USA))](http://redump.org/disc/81628/)
+[^108150]: [Case for TKKG, A - The Mystery of the Mayan Treasure (USA) (En,De)](http://redump.org/disc/108150/)
+[^48393]: [Emergency 4 - Global Fighters for Life (Germany) (Disc 1)](http://redump.org/disc/48393/)
+[^82475]: [Sluzhba 911 (Russia)][Remnants](http://redump.org/disc/82475/)
+[^DE_EM2DX]: [German Emergency 2 Deluxe](waiting on null to upload/)
+[^Petterson]: [Petterson und Findus - Hühnertanz und Katerzirkus](waiting on null to upload)
+[^Kenny]: [Kenny's Adventure](waiting on null to upload)
+[^101786]: [Max and the Magician (Europe) (En,Fr,De,Es)](http://redump.org/disc/101786/)
+
+#### copy-X light:
+
+Basic ring protection via dummy files intersected by rings. No software protection is used unless otherwise supplied by the game itself. Can be circumvented by simply not copying the dummy files when copying the rest of the game from the disc.
+
+#### copy-X professional:
+
+Builds upon copy-X light. Contains everything described there, but also adds software protection in the form of a disc check, which appears to additionally check one of the dummy files. Copy-X professional also checks if the contemporaneously-popular kernel debugging software SoftICE is active via a process called SoftICE-Test, checking both for "SoftICE for Windows 95" and for "SoftICE for Windows NT", ending game execution if detected when booting the game.
+
+#### Untested:
+
+* copy-X audio[^D_8232013][^D_1436646][^D_2182905][^D_451419]
```

Comment:
You don't really need the footnotes here since you don't need cite the fact that copy-X audio hasn't been tested yet. Since these same footnotes are already present in Samples anyway, just move the footnote creation to that section and remove the footnotes here.

Reply:
Fixed.

Reply:
Unfixed.

Reply:
Fixed.